jwst/coron/align_refs_step.py

```diff
@@ -38,6 +41,16 @@ def process(self, target, psf):
             # Open the input psf images
             psf_model = datamodels.open(psf)
 
+            # Retrieve the lenth of the smoothing box for the filter
+            try:
+                box_size = self.smoothing_length
+            except AttributeError:
+                box_size = 4
+                self.log.warning('Smoothing length not found, set to default')
+
+            # Smooth the psf images
+            psf_model = smooth_psf(psf_model, box_size)
```

Comment:
The smoothing operation needs to be applied to both the psf and target images, so this should also be called using `target_model` as input. And given that it's being applied to both sets of images (not just the PSF's) perhaps the name of the routine should be changed to something more generic than `smooth_psf`.